18,
 ['[2022-01-17 14:32:38,650] {standard_task_runner.py:89} ERROR - Failed to execute job 57 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:34:43,214] {standard_task_runner.py:89} ERROR - Failed to execute job 58 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:30:33,615] {standard_task_runner.py:89} ERROR - Failed to execute job 56 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:47:08,016] {standard_task_runner.py:89} ERROR - Failed to execute job 70 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:43:12,396] {standard_task_runner.py:89} ERROR - Failed to execute job 69 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:40:44,155] {standard_task_runner.py:89} ERROR - Failed to execute job 64 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:38:39,653] {standard_task_runner.py:89} ERROR - Failed to execute job 63 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:55:35,220] {standard_task_runner.py:89} ERROR - Failed to execute job 82 for task load_tesla_to_hdfs\n',
  '[2022-01-17 14:27:45,718] {standard_task_runner.py:89} ERROR - Failed to execute job 51 for task load_tesla_to_hdfs\n',
  '[2022-01-18 22:12:59,398] {standard_task_runner.py:89} ERROR - Failed to execute job 103 for task download_apple_data\n',
  '[2022-01-18 21:54:04,613] {standard_task_runner.py:89} ERROR - Failed to execute job 97 for task download_apple_data\n',
  '[2022-01-18 22:14:38,016] {standard_task_runner.py:89} ERROR - Failed to execute job 106 for task download_apple_data\n',
  '[2022-01-18 22:12:57,929] {standard_task_runner.py:89} ERROR - Failed to execute job 102 for task download_tesla_data\n',
  '[2022-01-18 21:54:03,098] {standard_task_runner.py:89} ERROR - Failed to execute job 96 for task download_tesla_data\n',
  '[2022-01-18 22:14:36,562] {standard_task_runner.py:89} ERROR - Failed to execute job 105 for task download_tesla_data\n',
  '[2022-01-18 22:10:29,148] {standard_task_runner.py:89} ERROR - Failed to execute job 99 for task create_data_directory\n',
  '[2022-01-18 22:12:30,613] {standard_task_runner.py:89} ERROR - Failed to execute job 100 for task create_data_directory\n',
  '[2022-01-18 22:08:25,953] {standard_task_runner.py:89} ERROR - Failed to execute job 98 for task create_data_directory\n']